tools/rp2040/Config.mk

```diff
@@ -26,17 +26,36 @@
 
 # POSTBUILD -- Perform post build operations
 
-ifeq ($(CONFIG_RP2040_UF2_BINARY),y)
-ifdef PICO_SDK_PATH
-define POSTBUILD
-	$(Q)echo "Generating: nuttx.uf2"; \
+PICOTOOL_BUILD_TEMP?=build_picotool
+PICOTOOL_BUILD_PATH?=$(PICO_SDK_PATH)$(DELIM)$(PICOTOOL_BUILD_TEMP)
```

Comment:
why not build inplace?

Reply:
> why not build inplace?

Currently already build in pico-sdk.

Reply:
I mean in source code instead build folder